openedx/core/djangoapps/theming/finders.py

```diff
@@ -35,21 +38,15 @@ class ComprehensiveThemeFinder(BaseFinder):
     def __init__(self, *args, **kwargs):
         super(ComprehensiveThemeFinder, self).__init__(*args, **kwargs)
```

Comment:
We should add a docstring here to describe what is happening in this initialization workflow